docs/htmlchecks/index.html

```diff
@@ -104,16 +104,16 @@ <h2>Results by Page </h2>
    <tbody>
       <tr>
         <td class="failures"><a href="#manualXhtml">manual.html</a></td>
-        <td class="number">808</td>
-        <td class="number">7</td>
+        <td class="number">689</td>
+        <td class="number">3</td>
         <td class="number failures">99%</td>
       </tr>
 
     </tbody>
   </table><hr>
 
 <h1 id="manualXhtml">Results for manual.html </h1>
-location : /workspace/docToolchain/build/docs/html5/manual.html <p>
```

Comment:
@clockworkorange I'm pretty sure you didn't want to change this one

Reply:
Sorry. Fixed